package.json

```diff
@@ -11,17 +11,15 @@
     "scriptoni": "./bin/scriptoni.js"
   },
   "scripts": {
-    "start": "babel -w src --out-dir lib",
-    "build": "rimraf lib/* && babel src --out-dir lib && npm run copy-files",
-    "copy-files": "babel-node src/copy-files.js",
-    "lint-style": "babel-node bin/scriptoni.js lint-style",
-    "lint-style-fix": "babel-node bin/scriptoni.js lint-style-fix",
-    "lint": "babel-node bin/scriptoni.js lint",
-    "metarpheus": "babel-node bin/scriptoni.js metarpheus",
-    "metarpheus-diff": "babel-node bin/scriptoni.js metarpheus-diff",
-    "test": "NODE_ENV= jest",
-    "preversion": "npm run lint && npm test",
-    "prepublish": "npm run build",
+    "build": "rimraf lib/* && tsc && yarn copy-files",
+    "copy-files": "ts-node ./copy-files.ts",
+    "lint-style": "node bin/scriptoni.js lint-style",
+    "lint-style-fix": "node bin/scriptoni.js lint-style-fix",
+    "metarpheus": "node bin/scriptoni.js metarpheus",
+    "metarpheus-diff": "node bin/scriptoni.js metarpheus-diff",
+    "test": "NODE_ENV= jest --runInBand",
```

Comment:
if i'm not wrong, without it `jest` sets `NODE_ENV` to `test`, but we can try removing it as see what happens

Reply:
yes, it's needed because `scriptoni` defaults `NODE_ENV` to `development` and `production` when running `web-build` and `web-dev` only if it's not defined and as `jest` sets it to `test` those script wouldn't work properly